src/stdlib/globals.md

```diff
@@ -39,3 +39,1303 @@ In addition to [low-level WebAssembly and compiler built-ins](./builtins.md), th
+  | i8  | i8x16.shr_s
+  | u8  | i8x16.shr_u
+  | i16 | i16x8.shr_s
+  | u16 | i16x8.shr_u
+  | i32 | i32x4.shr_s
+  | u32 | i32x4.shr_u
+  | i64 | i64x2.shr_s
+  | u64 | i64x2.shr_u
+  </details>
+
+* ```ts
+  function v128.and(a: v128, b: v128): v128
+  ```
+  Performs the bitwise `a & b` operation on each lane.
+
+* ```ts
+  function v128.or(a: v128, b: v128): v128
+  ```
+  Performs the bitwise `a | b` operation on each lane.
+
+* ```ts
+  function v128.xor(a: v128, b: v128): v128
+  ```
+  Performs the bitwise `a ^ b` operation on each lane.
+
+* ```ts
+  function v128.andnot(a: v128, b: v128): v128
+  ```
+  Performs the bitwise `!a & b` operation on each lane.
+
+* ```ts
+  function v128.not(a: v128): v128
+  ```
+  Performs the bitwise `!a` operation on each lane.
+
+* ```ts
+  function v128.bitselect(a: v128, b: v128, mask: v128): v128
+  ```
+  Selects bits of either vector according to the specified mask.
+
+* ```ts
+  function v128.any_true(a: v128): bool
+  ```
+  Reduces a vector to a scalar indicating whether any lane is considered `true`.
+
+* ```ts
+  function v128.all_true<T>(a: v128): bool
+  ```
+  <details><summary>Reduces a vector to a scalar indicating whether all lanes are considered <code>true</code>.</summary>
+
+  | T        | Instruction
+  |----------|-------------
+  | i8, u8   | i8x16.all_true
+  | i16, u16 | i16x8.all_true
+  | i32, u32 | i32x4.all_true
+  | i64, u64 | i64x2.all_true
+  </details>
+
+* ```ts
+  function v128.bitmask<T>(a: v128): bool
+  ```
+  <details><summary>Extracts the high bit of each integer lane (except 64-bit) and produces a scalar mask with all bits concatenated.</summary>
+
+  | T        | Instruction
+  |----------|-------------
+  | i8, u8   | i8x16.bitmask
+  | i16, u16 | i16x8.bitmask
+  | i32, u32 | i32x4.bitmask
+  | i64, u64 | i64x2.bitmask
+  </details>
+
+* ```ts
+  function v128.popcnt<T>(a: v128): v128
+  ```
+  <details><summary>Counts the number of bits set to one within each lane.</summary>
+
+  | T      | Instruction
+  |--------|-------------
+  | i8, u8 | i8x16.popcnt
+  </details>
+
+* ```ts
+  function v128.min<T>(a: v128, b: v128): v128
+  ```
+  <details><summary>Computes the minimum of each lane.</summary>
+
+  | T   | Instruction
+  |-----|-------------
+  | i8  | i8x16.min_s
+  | u8  | i8x16.min_u
+  | i16 | i16x8.min_s
+  | u16 | i16x8.min_u
+  | i32 | i32x4.min_s
+  | u32 | i32x4.min_u
+  | f32 | f32x4.min
+  | f64 | f64x2.min
+  </details>
+
+* ```ts
+  function v128.max<T>(a: v128, b: v128): v128
+  ```
+  <details><summary>Computes the maximum of each lane.</summary>
+
+  | T   | Instruction
+  |-----|-------------
+  | i8  | i8x16.max_s
+  | u8  | i8x16.max_u
+  | i16 | i16x8.max_s
+  | u16 | i16x8.max_u
+  | i32 | i32x4.max_s
+  | u32 | i32x4.max_u
+  | f32 | f32x4.max
+  | f64 | f64x2.max
+  </details>
+
+* ```ts
+  function v128.pmin<T>(a: v128, b: v128): v128
+  ```
+  <details><summary>Computes the psuedo-minimum of each lane.</summary>
```

Comment:
This one is taken right from the specification I believe because I couldn't find a proper explanation. Agree that we should explain it in more detail.